bot.js

```diff
@@ -1,12 +1,21 @@
-// IIFE (Immediatly Invoked Function Expression) to set up variables
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
dotenv is baked into repl.it, so this wont be needed

Reply:
True, although it might be useful if you ever want to run the bot on something other than repl.it

Reply:
Maybe just comment it out for future use.

Reply:
Could do. What about the dotenv requirement in package.json then? JSON can't be commented iirc.

Reply:
the package system is completely unneeded, repl.it just automatically adds that for some reason.

Reply:
also, this isn't meant as a package in the first place.

Reply:
package.json is actually extremely practical, if only so that whoever is locally testing the bot to make pull requests (like me for instance) doesn't have to manually read through the code to see which packages need installing, only to discover that they missed a few.

Reply:
The thing is that this isnt meant for that.